diff --git a/scripts/copy-language-pack.sh b/scripts/copy-language-pack.sh
index 7215efd..e506af6 100644
--- a/scripts/copy-language-pack.sh
+++ b/scripts/copy-language-pack.sh
@@ -5,12 +5,12 @@ copy_browser_locales() {
   mkdir -p $browser_locales/$langId
   if [ "$langId" = "en-US" ]; then
     find $browser_locales/$langId -type f -name "zen*" -delete
-    rsync -av --exclude=.git ./l10n/en-US/browser/ $browser_locales/$langId/
+    rsync -a --exclude=.git ./l10n/en-US/browser/ $browser_locales/$langId/
     return
   fi
   rm -rf $browser_locales/$langId/
   # TODO: Copy the rest of the l10n directories to their respective locations
-  rsync -av --exclude=.git ./l10n/$langId/ $browser_locales/$langId/
+  rsync -a --exclude=.git ./l10n/$langId/ $browser_locales/$langId/
 }
 
 LANG=$1
diff --git a/scripts/download-language-packs.sh b/scripts/download-language-packs.sh
index 7fa7047..13ae401 100644
--- a/scripts/download-language-packs.sh
+++ b/scripts/download-language-packs.sh
@@ -1,5 +1,5 @@
 
-set -ex
+set -e
 
 CURRENT_DIR=$(pwd)
 
@@ -19,7 +19,7 @@ update_language() {
 
   echo "Updating $langId"
   # move the contents from ../firefox-l10n/$langId to ./l10n/$langId
-  rsync -av --progress ../firefox-l10n/$langId/ . --exclude .git
+  rsync -a ../firefox-l10n/$langId/ . --exclude .git
 
   cd $CURRENT_DIR
 }
